samples_project/Assets/SampleViewer/Samples/FeatureLayer/ScrollViewItem.cs

```diff
@@ -0,0 +1,67 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using Esri.ArcGISMapsSDK.Components;
+using TMPro;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using UnityEngine.UI;
+
+public struct Data
```

Comment:
This should be something specific, `Data` is too generic and could name clash with something else